### PR TITLE
fix(docker): prevent creation of new files via touch

### PR DESCRIPTION
**Pull request by petrjasek**
if there was a file with space in name it would split it
for xargs and touch every part of that filename, creating new files,
and not touching the original file at all

but not sure why it's there @actionless ?

### CODE REVIEW

**Comment by actionless** [comment]
it's to reset mtime

File: docker/container_build.sh [hunk 1]
```diff
@@ -39,9 +39,9 @@ mkdir -p $SCREENSHOTS_DIR
 
 # reset repo files' dates:
 cd $BAMBOO_DIR/server/
-find ./ | grep -v .git/ | xargs touch -t 200001010000.00
+find -print0 ./ | grep -vzZ .git/ | xargs -0 touch -t 200001010000.00
 cd $BAMBOO_DIR/client/
-find ./ | grep -v .git/ | xargs touch -t 200001010000.00
+find -print0 ./ | grep -vzZ .git/ | xargs -0 touch -t 200001010000.00
```
Comment (actionless) [inline docker/container_build.sh]: actually it can be deduplicated, ie `cd` to whole `$BAMBOO_DIR` instead of each client and server

**Comment by actionless** [comment]
LGTM

**Comment by amagdas** [comment]
:+1:

**Pull request merged**